src/main/webapp/js/finscholar/commonlistview/commonlistview.js

```diff
@@ -52,13 +52,14 @@ class CommonListView {
   async init(tableContainer) {
     tableContainer.innerHTML = commonlistview({pageIndex: this.optionIndex_});
     this.container_ = googDom.getElement(ITEM_CONTAINER_ID)
-    window.addEventListener('scroll', await this.bindedScrollHandler_);
+    window.addEventListener('scroll', this.bindedScrollHandler_);
```

Comment:
Was it necessary for this part to be asynchronous?

Reply:
Actually no. The handler itself is asynchronous anyways lol.